src/renderer/directives/vModelComboBox.ts

```diff
@@ -0,0 +1,22 @@
+import { ObjectDirective } from '@vue/runtime-core';
+import { VNComboBox } from '../../widgets/ComboBox/VNComboBox';
+
+type ModelDirective<T> = ObjectDirective<T & { _assign: Function }>
+
+export const vModelComboBox: ModelDirective<VNComboBox> = {
+  beforeMount: (el, { value }, vnode) => {
+    el.setCurrentIndex(value);
+    // eslint-disable-next-line no-param-reassign, no-underscore-dangle
+    el._assign = vnode.props!['onUpdate:modelValue'] as Function;
+    el.addEventListener('currentIndexChanged', (indexValue) => {
+      // eslint-disable-next-line no-underscore-dangle
+      el._assign(indexValue);
```

Comment:
Used the currentIndex value for implementing v-model – would this be okay?

Reply:
Yeah, index feels safer than the text event.